tests/Core/Tokenizer/ExplicitOctalNotationTest.inc

```diff
@@ -0,0 +1,4 @@
+<?php
+
+/* testExplicitOctal declaration */
+$foo = 0o137041;
```

Comment:
Again: what about a number with uppercase `o` ?

Reply:
Also wondering if any test could be conceived where the token stream should _not_ be changed (i.e. guard against false positives).